Formula/qscintilla2.rb

```diff
@@ -12,14 +12,17 @@ class Qscintilla2 < Formula
   end
 
   option "without-plugin", "Skip building the Qt Designer plugin"
+  option "without-python", "Skip building the python bindings"
 
   depends_on :python => :recommended
```

Comment:
> Where should this stuff go in case we're not pulling in Homebrew's Python?

It should always go into the Cellar.

> Should the build system be able to deal with that or do we have to go back to a brewed Python?

The build system must be able to deal with it; right now this is badly broken.